app/screens/AppNavigator/screens/Dex/DexRemoveLiquidity.tsx

```diff
@@ -70,20 +73,47 @@ export function RemoveLiquidityScreen (props: Props): JSX.Element {
       .catch(Logging.error)
   }, [])
 
+  useEffect(() => {
+    setValidity(
+      Number(percentage) > 0 &&
+      Number(percentage) <= 100 &&
+      !hasPendingJob
+    )
+  }, [percentage])
+
   return (
     <ScrollView style={tailwind('w-full flex-col flex-1 bg-gray-100')}>
       <View style={tailwind('w-full bg-white mt-8')}>
-        <View style={tailwind('w-full flex-row p-4')}>
+        <View style={tailwind('w-full flex-row p-4 items-stretch')}>
           <Text
-            style={tailwind('flex-1 font-semibold')}
+            style={tailwind('w-2/4 font-semibold flex-1')}
           >{translate('screens/RemoveLiquidity', 'Amount to remove')}
           </Text>
-          <Text testID='text_slider_percentage' style={tailwind('text-right')}>{percentage}%</Text>
+          <NumberTextInput
+            testID='text_input_percentage'
+            style={[
+              tailwind('text-right w-2/4 p-0 mr-0.5'),
+              Platform.OS === 'ios' && tailwind('-mt-0.5'),
```

Comment:
store this on a boolean instead of inline, so it's easier for context

Reply:
you mean this `Platform.OS === 'ios'` right? Should i refactor to a constant? So it will be available project-wide

Reply:
no no, just put it as a const above the return. `isIOS` then use that boolean here instead of inlining the expression

Reply:
yes but i noticed the same inline code is used in HeaderTitle too, should we update that?